test/scrabble_test.rb

```diff
@@ -6,8 +6,21 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
+
     assert_equal 4, Scrabble.new.score("f")
   end
+
```

Comment:
Make sure to add tests for every piece of functionality (you don't have any for whole words).